### PR TITLE
added missing preParsing properties

### PR DESCRIPTION
**Pull request by ionutale**
fix for this issue :
https://github.com/fastify/help/issues/424

poc with the issue:
https://github.com/ionutale/poc-fastify-preParsing-error

### CODE REVIEW

**Comment by mcollina** [comment]
Can you please add a unit test?

File: lib/context.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const { kFourOhFourContext, kReplySerializerDefault } = require('./symbols.js')
 
 // Objects that holds the context of every request
 // Every route holds an instance of this object.
-function Context (schema, handler, Reply, Request, contentTypeParser, config, errorHandler, bodyLimit, logLevel, logSerializers, attachValidation, replySerializer, schemaErrorFormatter) {
+function Context(schema, handler, Reply, Request, contentTypeParser, config, errorHandler, bodyLimit, logLevel, logSerializers, attachValidation, replySerializer, schemaErrorFormatter) {
```
Comment (mcollina) [inline lib/context.js:7]: why this change?

File: lib/context.js [hunk 2]
```diff
@@ -28,7 +31,7 @@ function Context (schema, handler, Reply, Request, contentTypeParser, config, er
   this.schemaErrorFormatter = schemaErrorFormatter || defaultSchemaErrorFormatter
 }
 
-function defaultSchemaErrorFormatter (errors, dataVar) {
+function defaultSchemaErrorFormatter(errors, dataVar) {
```
Comment (mcollina) [inline lib/context.js:34]: please restore this

**Review by mcollina — CHANGES_REQUESTED** [review]

File: lib/context.js (hunk 1 above)
Comment (ionutale) [reply lib/context.js]: This was the theia IDE that formatted the code upon saving.

**Comment by ionutale** [comment]
> Can you please add a unit test?

yes, i'm working on it.

**Comment by ionutale** [comment]
i will come back with a new pull request, that will have the change and the tests, without all of those noise commits

**Pull request closed without merging**